mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -797,7 +797,7 @@ def Rock_BlockwiseGemmOp:
                    MemRefRankOf<[F32, F16, BF16, I8], [3]>:$matrixB,
                    MemRefRankOf<[F32, F16, BF16, I32], [2]>:$matrixC,
                    I32Attr:$blockSize,
-                   Rock_GeneralGemmParamsAttr:$params
+                   OptionalAttr<RockTuningParamAttrInterface>:$params
```

Comment:
(sorry for comments in backwards) This is the non-xdlops branch, it requires the non-xdlops params